langlib/lang.query/src/main/ballerina/helpers.bal

```diff
@@ -126,7 +126,7 @@ function toString(stream<Type, error?> strm) returns string {
 function addToTable(stream<Type, error?> strm, table<map<Type>> tbl, error? err) returns table<map<Type>>|error {
     record {| Type value; |}|error? v = strm.next();
     while (v is record {| Type value; |}) {
-        error? e = trap tbl.add(<map<Type>> v.value);
+        error? e = trap tbl.add(<map<Type>> check v.value);
```

Comment:
Here again, to preserve the previous behaviour, we need to use `checkpanic`? Please check other changes too.

Reply:
Fixed and checked the other places too.